src/main/java/com/joss/conductor/mobile/ConductorConfig.java

```diff
@@ -248,20 +248,20 @@ public void setAppiumVersion(String appiumVersion) {
         this.appiumVersion = appiumVersion;
     }
 
-    public int getTimeout() {
-        return timeout;
+    public int getAppiumRequestTimeout() {
```

Comment:
Oh yeah this is the other breaking change: `getDriver().configuration.getRetries()` will need to be refactored, so maybe here we can actually deprecate this, or just keep the same name `getDriver().configuration.getRetries()`  and have it still `return appiumRequestTimeout;`

^ This would limit the breaking changes to just `config.yaml` changes needed.

Also, I'm kind of unsure if even allowing the user to `getDriver().configuration.getRetries()`  is a good thing.  Perhaps `getRetries` does not need to be public anymore with the changes to `isPresentWait` and `waitForElement` to not use `configuration.getRetries()`

Just for documentation and readers without context, a big problem with Conductor stems from many of its methods using `getRetries` under the hood, without user awareness, to control a `Thread.sleep` loop. This behavior was not configurable, it was hidden and the ambiguity of the naming (which this PR fixed) made the consequences of those two things even worse!

Reply:
3 options:

1) Keep `getTimeout()` but deprecate it and have it `return appiumRequestTimeout` *AND* make `getAppiumRequestTimeout()`
2) Only use `getTimeout` and have it `return appiumRequestTimeout()`
3) Keep what we have now and list this as another thing that needs to be changed in the Release Notes for Breaking Changes, also potentially warn user that this is a bad practice to begin with

Reply:
I think going the deprecation route is the safest way
Here's my first attempt at deprecating the methods:
```
/***
     * Returns the {@link #timeout} variable, which has been deprecated in favor of {@link #appiumRequestTimeout} for clarity
     *
     * @deprecated use {@link #getAppiumRequestTimeout()} instead
     */
    @Deprecated(since = "0.19.1")
    public int getTimeout() {
        return timeout == 0 ? appiumRequestTimeout : timeout;
    }

    public int getAppiumRequestTimeout() {
        return appiumRequestTimeout;
    }

    /***
     * Sets the {@link #timeout} variable, which has been deprecated in favor of {@link #appiumRequestTimeout} for clarity
     * Consider replacing the `timeout:` options in the config yaml with `appiumRequestTimeout`
     *
     * @deprecated use {@link #setAppiumRequestTimeout(int)} instead
     */
    @Deprecated(since = "0.19.1")
    public void setTimeout(int timeout) {
        this.timeout = timeout;
    }

    public void setAppiumRequestTimeout(int appiumRequestTimeout) {
        this.appiumRequestTimeout = appiumRequestTimeout;
    }

    /***
     * Returns the {@link #retries} variable, which has been deprecated in favor of {@link #implicitWaitTime} for clarity
     *
     * @deprecated use {@link #getImplicitWaitTime()} instead
     */
    @Deprecated(since = "0.19.1")
    public int getRetries() {
        return retries == 0 ? implicitWaitTime : retries;
    }

    public int getImplicitWaitTime() {
        return implicitWaitTime;
    }

    /***
     * Sets the {@link #retries} variable, which has been deprecated in favor of {@link #implicitWaitTime} for clarity
     * Consider replacing the `retries:` options in the config yaml with `implicitWaitTime`
     *
     * @deprecated use {@link #setImplicitWaitTime(int)}  instead
     */
    @Deprecated(since = "0.19.1")
    public void setRetries(int retries) {
        this.retries = retries;
    }

    public void setImplicitWaitTime(int implicitWaitTime) {
        this.implicitWaitTime = implicitWaitTime;
    }
```

Reply:
Additionally, I re-added the deprecated variables and marked them as deprecated, but did not initialize them. I added logic on the getters that if the old variable is 0 (meaning it hasn't been initialized), we instead return the new variables. The logic behind this is that the variable is 0 would only occur when the variable isn't set on their custom yaml, and that we should now force them into using the new variables.

Reply:
Spoke to @evant and made some slight modifications to the above, added in 260ddece1693c86288ea46113614d9e6c548c127